tldw_Server_API/tests/Billing/test_authnz_metering_repository.py

```
@@ -0,0 +1,163 @@
+from __future__ import annotations
+
+import sqlite3
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+
+class _AcquireContext:
+    def __init__(self, conn):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)